### PR TITLE
cfitsio 3.470

### PR DESCRIPTION
**Pull request by devonhollowood**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by devonhollowood** [comment]
If this ends up needing revision bumps (like #41705 and #41994 before it), I'll do those either tonight or tomorrow.

File: Formula/cfitsio.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Cfitsio < Formula
   desc "C access to FITS data files with optional Fortran wrappers"
   homepage "https://heasarc.gsfc.nasa.gov/docs/software/fitsio/fitsio.html"
-  url "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio3450.tar.gz"
-  version "3.450"
-  sha256 "bf6012dbe668ecb22c399c4b7b2814557ee282c74a7d5dc704eb17c30d9fb92e"
-  revision 1
+  url "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-3.47.tar.gz"
+  version "3.470"
```
Comment (javian) [inline Formula/cfitsio.rb]: Considering the file name changed for the release tar it might be that this is now detected automatically.

**Comment by SMillerDev** [comment]
@devonhollowood can you resolve the merge conflict?

**Comment by devonhollowood** [comment]
@SMillerDev Done.

**Comment by devonhollowood** [comment]
I added a revision bump for `vips`. It looks to me like the failures for `astrometry-net` and `mapserver` are spurious, but let me know if that's not the case.

**Comment by Bo98** [comment]
Thanks for this. We'll retest this soon and see what the issue is.

We're currently building Python 3.8 (see #45337), which involves revision bumping almost all Python formulae, so we're going to wait on this until that's finished to avoid merge conflicts. I'll let you know when we're ready.

**Comment by fxcoudert** [comment]
@devonhollowood sorry for dropping the ball on this one, can you please reopen a new PR rebased on top of master? There are several conflicts now :(

**Pull request closed without merging**